src/schemas/ProgramDonorSummary/resolvers.ts

```diff
@@ -7,25 +7,18 @@ const programDonorSummaryEntriesResolver: GraphQLFieldResolver<
   unknown,
   GlobalGqlContext,
   {
-    programId: string;
+    programShortName: string;
     first: number;
     last: number;
     filters: ProgramDonorSummaryFilter[];
   }
 > = (source, args, context): DonorSummaryEntry[] => {
-  const { programId } = args;
+  const { programShortName } = args;
 
   console.log('args: ', args.filters);
 
   return [];
 };
-
-const totalDonorsCountResolver: GraphQLFieldResolver<
```

Comment:
couldn't we delete the other donorCount resolvers too?

Reply:
yes we can! sorry forgot to, and already did that in another coming PR, so will skip for this PR :) thanks!